### DISCUSSION TITLE
Jeka : a build tool that integrates nicely with Fabric8

### DISCUSSION THREAD

**Discussion by djeang** [fabric8io/kubernetes-client#5317] [Show and tell]
Hi,

I am the owner of [Jeka Project](http://jeka.dev) which is a Java build tool that integrates nicely with third party libraries or tools.

With this [demo](https://github.com/jeka-dev/working-examples/tree/master/kubernetes), I want to showcase, how it can be easy to build a spring-boot application and deploy it to deploy it to K8s without the need of a specific plugin : just using Jeka along Jib and Fabric8 core libraries.

This demo also handles CI/CD cases where Kubernetes resources slightly differ from an environment to another.

I'll be glad to receive some feedbacks . I am pretty new to Fabric8 and interested on how you would handle this case differently.

Jérôme

**Comment by manusa** [comment]
Hey @djeang,
This looks nice.
We're also maintaining [Eclipse JKube](https://github.com/eclipse/jkube).
I think it might be interesting to provide an example using Eclipse JKube (jkube-kit) and Jeka, which would be a much more simple alternative to the Fabric8 Kubernetes Client example you already have.
If I find the time I'll try to submit a PR ;)
